### PR TITLE
Refactor dashboard layout component

### PR DESCRIPTION
**Pull request by clhenrick**
Allows for configuring the Dashboard for three possible layouts: Chart, Map, or Chart & Map via tabs.

### CODE REVIEW

File: src/components/tools/Partials/Dashboard/View.svelte [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<script>
+  export let useTabs = false;
+</script>
+
+<!-- Chart slots -->
+{#if $$slots.chart_title}
```
Comment (clhenrick) [inline src/components/tools/Partials/Dashboard/View.svelte:6]: TIL that `$$slots` is a special variable that is an object with the name of slots being passed to a component ([link to docs](https://svelte.dev/docs#template-syntax-slot-$$slots)). It's useful for only rendering a slot or other content if the parent component uses the slot.

File: src/components/tools/Partials/Dashboard/index.svelte [hunk 2]
```diff
@@ -2,13 +2,18 @@
...
+
   export let useTabs = false;
+  export let useMap = false;
   export let activeTab = 0;
 
   const activeTabDispatcher = createEventDispatcher();
 
   $: activeTab, dispatchActiveTab();
 
+  $: console.log($$slots);
```
Comment (clhenrick) [inline src/components/tools/Partials/Dashboard/index.svelte]: rm console.log

File: src/components/tools/Partials/Dashboard/index.svelte [hunk 3]
```diff
@@ -100,78 +106,45 @@
...
-
-                  <div class="bx--row">
-                    <div class="bx--col">
-                      <slot name="tab_content_slippy_map_controls">
-                        tab_content_slippy_map_controls
-                      </slot>
-                    </div>
-                  </div>
+                  <View useTabs="{true}">
+                    <slot slot="map_title" name="map_title" />
```
Comment (clhenrick) [inline src/components/tools/Partials/Dashboard/index.svelte:108]: This syntax looks a little odd but it's how you forward slotted content to a child component from a parent component.

**Comment by clhenrick** [comment]
@mukhtyar this will allow for the Coastal Flooding SLR tool to not have the chart / map tabs. It will also support future map focused tools.

**Review by mukhtyar — APPROVED** [review]

**Pull request merged**